### ISSUE TITLE
Env var OCI_CONFIG_PROFILE is not being honored

### ISSUE THREAD

**Issue by plakdawa** [oracle/oci-ansible-modules#28]
## Issue Report

**Describe the issue**

The ~/.oci/config file is present and properly configured. The env var OCI_CONFIG_PROFILE is set. However, running a simple playbook throws the following error:

``fatal: [localhost]: FAILED! => {"changed": false, "msg": "Invalid OCI configuration. Exception: {'user': 'missing', 'key_file': 'missing', 'fingerprint': 'missing', 'tenancy': 'missing'}"}``

Explicitly creating the following variable and using it in the module makes everything work fine.

* ``config_profile_name: "{{ lookup('env', 'OCI_CONFIG_PROFILE') }}"`` 

**Expected behavior**

The module should automatically look at the ``OCI_CONFIG_PROFILE`` env var.

**Environment**

* OS version: ``Oracle Linux Server release 7.4``

* Ansible version:

``
ansible 2.7.8
  config file = None
  configured module search path = ['/home/plakdawa/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /opt/peo-oci-toolkit/miniconda3-4.5.12/envs/py3forocidev/lib/python3.6/site-packages/ansible
  executable location = /opt/peo-oci-toolkit/miniconda3-4.5.12/envs/py3forocidev/bin/ansible
  python version = 3.6.8 |Anaconda, Inc.| (default, Dec 30 2018, 01:22:34) [GCC 7.3.0]
``

* OCI Python SDK version:

    ``2.1.4``

* OCI Ansible Modules version:

    ``1.5.0``

**Ansible playbook to reproduce the issue**
```
---
- name: Create public IP
  hosts: localhost
  vars:
    config_profile_name: "{{ lookup('env', 'OCI_CONFIG_PROFILE') }}"
    my_label:
    sandbox_compartment_ocid: "{{ lookup('env', 'sandbox_compartment_ocid') }}"
  gather_facts: no
  tasks:
    - name: Create a public IP with lifetime as RESERVED
      oci_public_ip:
        #Comment/Uncomment the following line to reproduce/resolve the issue
        #config_profile_name: "{{ config_profile_name }}"
        compartment_id: "{{ sandbox_compartment_ocid }}"      
        display_name: "{{ my_label }}_public_ip"
        lifetime: RESERVED
      register: result
    - debug: msg="test_public_ip = {{ result }}"
```

**Comment by sivakumart** [comment]
Thanks @plakdawa for raising this. There appears to be a bug in the way we handle overrides for config profile overrides. We will fix this issue in an upcoming release, and update this issue.

**Comment by manojmeda** [comment]
@plakdawa This issue has been addressed in the latest release (`v1.6.0`). Could you please check and let us know if you face any issues.

**Comment by plakdawa** [comment]
@manojmeda Confirming that this is fixed. Thanks a lot for the quick response.
